storage/replica_command.go

```diff
@@ -125,6 +125,7 @@ func (r *Replica) executeCmd(ctx context.Context, raftCmdID storagebase.CmdIDKey
 		var resp roachpb.EndTransactionResponse
 		resp, intents, err = r.EndTransaction(batch, ms, h, *tArgs)
 		reply = &resp
+		log.Infof("END TXN: %s: %s", reply.Header().Txn, err)
```

Comment:
Still debugging very infrequently-occurring runs of `acceptance/TestSingleKey`; ignore these log statements.